relayer-cli/src/commands/keys.rs

```diff
@@ -16,6 +17,10 @@ pub enum KeysCmd {
     #[options(help = "Adds a key to a configured chain")]
     Add(add::KeysAddCmd),
 
+    /// The `keys clean` subcommand
+    #[options(help = "Removes a key from a configured chain")]
+    Clean(clean::KeysCleanCmd),
```

Comment:
Should we name the command delete instead of clean? Just to stay consistent with `gaiad`?

Reply:
I agree. Additionally, `delete` would be more accurate than `clean`.